lib/tbot/tbot_test.go

```diff
@@ -81,11 +81,11 @@ type defaultBotConfigOpts struct {
 
 func defaultTestServerOpts(t *testing.T, log *slog.Logger) testenv.TestServerOptFunc {
 	return func(o *testenv.TestServersOpts) {
-		testenv.WithClusterName(t, "root.localhost")(o)
+		testenv.WithClusterName(t, "root")(o)
```

Comment:
Does the Cluster Name need to be changed? Or is it just for convenience?

Reply:
Just for convenience - I figured no point having it be `root.localhost`, if we're not actually going to use that as the address for the proxy etc.